src/Shared/BrowserTesting/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,58 @@
+Microsoft.AspNetCore.BrowserTesting.BrowserKind
```

Comment:
Are we sure this is something we want to signup to own publicly as opposed to just something we use for our own tests right away?

Reply:
Is the package not marked as non-shipping?

In any case I want to make sure we don't change the APIs here lightly

Reply:
Ah ok, I just assumed we only did these public api checks for shipping packages

Reply:
We need to do the piece for saving the "artifacts", so we can make any changes needed.

Reply:
> Is the package not marked as non-shipping?

This project doesn't create a package at all. That said, I don't understand why we want to make changes in the project more difficult or (for now, due to the VS requirement) limited to Windows

Reply:
I've removed it for now, but I'll be enabling it in the future. I don't want folks making changes without being careful, this might become something similar to Microsoft.AspNetCore.Mvc.Testing in the future